extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -742,20 +740,22 @@ private static <DATA_TYPE> boolean tryEncodeDictionary(
 
             columnWriter.addDictionaryPage(encodedKeys, keyCount);
             final Iterator<IntBuffer> arraySizeIt = arraySizeBuffers == null ? null : arraySizeBuffers.iterator();
+            // Accumulate integer statistics in a temporary object.
```

Comment:
I am assuming this is needed because we are generating stats over the keys here, mainly to get stats of the nulls, which we then add to overall stats.
Is that right?
Can you please add a more descriptive comment to explain that what we mean by integer statistics here?